tests/test_mock_controller.py

```diff
@@ -82,17 +83,17 @@ async def test_missing_command_parameter(self):
         self.assertEqual(self.data["timeout"], -1)
 
     async def prepare_amcs(
-        self, initial_position, target_azimuth, target_rate,
+        self, initial_position, target_position, target_velocity,
     ):
         """Utility method for preparing the in initial state of AMCS for easier testing.
```

Comment:
"preparing the in initial state"? Is there an extra "in"? If not, might you reword for clarity?

Reply:
Extra "in" removed.